src/Entity/Field.php

```diff
@@ -1,6 +1,4 @@
-<?php
-
-declare(strict_types=1);
+<?php declare(strict_types=1);
```

Comment:
Why all the strict type declarations are moved to the first line? Please fix it (PSR12)